src/handlers/topic1.py

```
@@ -0,0 +1,29 @@
+class Topic1Handler(BaseHandler):
+    
+    async def _process_message(self, message: MessageEnvelope) -> bool:
+        try:
+            logger.info(f"Processing message: {message.to_dict()}")
+            # Add your actual message processing logic here
+            user = User(**message.body)
+            logger.info(f"Found user in message: {user.model_dump_json()}")
+            # return True to commit offset
+            return True
+        except ValidationError as e:
+            raise NonRetryableError(e)
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)